limes/resources/report_project.go

```diff
@@ -71,11 +71,16 @@ type ProjectAZResourceReport struct {
 	PlannedCommitments map[string]uint64 `json:"planned_commitments,omitempty"`
 	Usage              uint64            `json:"usage"`
 	PhysicalUsage      *uint64           `json:"physical_usage,omitempty"`
-	MinHistoricalUsage uint64            `json:"min_historical_usage,omitempty"`
-	MaxHistorcialUsage uint64            `json:"max_historical_usage,omitempty"`
+	HistoricalUsage    *HistoricalReport `json:"historical_usage,omitempty"`
 	Subresources       json.RawMessage   `json:"subresources,omitempty"`
 }
 
+type HistoricalReport struct {
+	MinUsage uint64 `json:"min_usage,omitempty"`
+	MaxUsage uint64 `json:"max_usage,omitempty"`
+	Duration string `json:"duration,omitempty"`
```

Comment:
A structured type exists for this:
```suggestion
	Duration CommitmentDuration `json:"duration,omitempty"`
```